Docs/src/tutorials/basictutorials/basictutorial1.md

```diff
@@ -0,0 +1,347 @@
+
+```cpp
+Ogre::Entity* ogreEntity4 = scnMgr->createEntity("ogrehead.mesh");
+
+Ogre::SceneNode* ogreNode4 = scnMgr->getRootSceneNode()->createChildSceneNode();
+ogreNode4->setPosition(-84, 48, 0);
+ogreNode4->roll(Ogre::Degree(-90));
+ogreNode4->attachObject(ogreEntity4);
+```
+
+Compile and run your application. We should now have a rotated Ogre head in our scene.
+
+![](bt1_rotated_entity.png)
+
+## The Ogre Environment
+
+The library and configuration files for Ogre can be found in the 'bin' folder of your OgreSDK. You should use the debug files when building your application in debug mod
+
+### Libraries and Plugins
+
+Ogre is divided into three shared library groups: main library, plugins, and third-party libraries.
+
+#### Main library
+
+The main library group contains the Ogre library itself and the shared libraries it relies on. The Ogre library is contained within OgreMain.dll or libOgreMain.so depending on your platform. This library must be included in all of your Ogre applications. OgreMain.dll requires a few other libraries like cg.dll.
+
+#### Plugins
+
+The second group of shared libraries are the plugins. Ogre pushes a good portion of its functionality into shared libraries so that they may be turned on or off easily. The core plugins that are included with Ogre have names that start with "Plugin_". You can also write your own plugins.
+
+Ogre also uses plugins for the different render systems (such as OpenGL, DirectX, etc). These plugins start with "RenderSystem_". This is also so that you can add only the systems you will need. This can be useful if you write shaders that rely on a particular system, because you can simply remove the incompatible system so that the program won't try to run incorrect code. This also means you can write your own plugins if you want to extend Ogre into another render system.
+
+#### Third-party Plugins
+
+The last major group contains third-party libraries and other general support libraries. Ogre is focused sharply on being a graphics rendering library. This group makes it easy to integrate external libraries to add things like physics, input, and GUI systems. These libraries are used together to form a full game development environment. You might find this piecemeal approach a little strange, but it is a very common design pattern in large software projects. It is harder to comprehend at first, but it is a much more flexible approach when you want to start building more complicated scenes.
+
+The Ogre demos and SDK include some of these third-party libraries. The [Open Input System](http://www.ogre3d.org/tikiwiki/tiki-index.php?page=OIS) can be used to manage input events and distribute them to Ogre. This is contained in OIS.dll or libOIS.so. You can also make use of Cg, which is used by CgProgramManager. This library allows you to produce materials with custom shaders. There are other libraries (not included with Ogre) that offer functionality such as sound and physics.
```

Comment:
with 1.10 we have replaced OIS by SDL2

Reply:
Does it mean that we should remove any notion of OIS in tutorials at all? 

Basic Tutorial 6
http://www.ogre3d.org/tikiwiki/tiki-index.php?page=Basic+Tutorial+6&structure=Tutorials

has full source (which built application from from scratch) and it works on 1.10. In this example OIS is used as input system. So I was expecting to left room for this part as well so that learner can see that Ogre is able to work not only with SDL as an input.

Reply:
> Does it mean that we should remove any notion of OIS in tutorials at all?

yes, OIS is dead. If anything I would make an example for Qt. But we should focus on the core parts first..

Reply:
I see. Then will redo everything in SDL input context.